typings/rawDataTypes.d.ts

```diff
@@ -279,3 +290,52 @@ export interface APIApplicationRoleConnectionMetadata {
   description: string;
   description_localizations?: LocalizationMap;
 }
+
+export interface APITextBasedChannel<T extends ChannelType> extends APIChannelBase<T> {
```

Comment:
Can you please make this change in a different PR? Its not really tied to this PR

Reply:
if you really want to, okey